StRoot/StarGenerator/macros/production/simulation.y2014x.pythia8.HFjets.input.list

```diff
@@ -0,0 +1,10000 @@
+/star/simu/simu/gstardata/Simu2022/rcf22000/vertex_files/15165031_0014
```

Comment:
Is it possible to run the submitted macros as is without regenerating these files?

Reply:
Those files are an input dependency to the job.  The format of the files are:

Run#   Event#   Vx   Vy   Vz...

Updating the macro with the specification.   These files would need to be regenerated when comparing against any production of the data... (any change which alters vertex finding would require a new set of inputs).

Reply:
If you explain the format of the input files and how to regenerate them then there is not much point in including this particular list which, as you say, need to be regenerated for other data anyway.